components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/model/AuthenticatedUser.java

```diff
@@ -164,6 +168,38 @@ public static AuthenticatedUser createLocalAuthenticatedUserFromSubjectIdentifie
         return authenticatedUser;
     }
 
+    /**
+     * Resolve userId from ancestor organizations when user is not found in the current organization.
+     *
+     * @return return user Id.
+     */
+    private String resolveUserIdFromAncestorOrganizations() {
+
+        try {
+            if (authenticatedSubjectIdentifier != null) {
+                String organizationId = FrameworkServiceDataHolder.getInstance().getOrganizationManager()
+                        .resolveOrganizationId(tenantDomain);
+                /*
+                    Extract user id from the authenticated subject identifier assuming user ID set as a part of the
+                    subject identifier.
+                */
+                String mayBeUserId = authenticatedSubjectIdentifier.split(UserCoreConstants.TENANT_DOMAIN_COMBINER)[0];
+                if (mayBeUserId.contains(CarbonConstants.DOMAIN_SEPARATOR)) {
+                    mayBeUserId = mayBeUserId.split(CarbonConstants.DOMAIN_SEPARATOR)[1];
+                }
+                Optional<String> userResideOrgId = FrameworkServiceDataHolder.getInstance()
+                        .getOrganizationUserResidentResolverService()
+                        .resolveResidentOrganization(mayBeUserId, organizationId);
+                if (userResideOrgId.isPresent()) {
+                    return mayBeUserId;
+                }
+            }
+        } catch (OrganizationManagementException e) {
+            log.debug("Error while resolving the user id from ancestor organizations.");
```

Comment:
```suggestion
            log.debug("Error while resolving the user id " +  mayBeUserId + "  from ancestor organizations of organization " +  organizationId);
```
Lets add more detail into the debug logs

Reply:
Fixed by - https://github.com/wso2/carbon-identity-framework/commit/2b5f8964d9c4d2ee2bcddaca01f44c59642bbf80

Also changed it to a error log instead of debug log.